spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/amqp/RabbitProperties.java

```diff
@@ -112,6 +113,11 @@
 
 	private List<Address> parsedAddresses;
 
+	/**
+	 * Enable metrics.
+	 */
+	private boolean metrics = true;
```

Comment:
That looks the wrong  place to  do this. Metrics related properties are in `management.metrics`. Check other implementations for inspiration (the JDBC one for  instance).

Reply:
Would `management.metrics.rabbitmq.instrument-connection-factory` work?

Reply:
We don't need a specific enablement property for Rabbit, as we have property-bound `MeterFilter`s in Boot 2. For example, if all of the Rabbit metrics are prefixed with "rabbit", then `management.metrics.filter.rabbit.enabled=false` would disable them.